gdk-pixbuf/Gir.toml

```diff
@@ -49,19 +49,10 @@ status = "generate"
     #unimplementable, use get_pixels instead
     ignore = true
     [[object.function]]
-    name = "new_from_file"
-    manual = true
-    [[object.function]]
     name = "new"
         [object.function.return]
         nullable = true
```

Comment:
I.e. this needs to be `false` and also for the other two functions

Reply:
okay